### ISSUE TITLE
Allow indivdual users to include content from other instances in the Explore feed

### ISSUE THREAD

**Issue by seanthegeek** [mastodon/mastodon#22410]
### Pitch

Allow the user to combine the public Explore feeds of other servers into their personal Explore feed, and allow the user to directly interact with posts the Explore feeds of other instances from their instance. Allow admins to include specific server feeds to include by default.

### Motivation

Smaller instances/servers may not have much interesting content in their Explore feed compared to larger ones. The Federated feed is way too active/noisy to find interesting content from other servers. The current process of visiting another instance's public Explore page in a new tab/window, clicking on a post you want to interact with, copying the URL, pasting it in the search of your server, and clicking on the post in the search results in order to interact with a singe post on another server is very cumbersome.

**Comment by nemobis** [comment]
> Allow the user to combine the public Explore feeds of other servers

Do you mean, to show content which is currently not federated, like https://github.com/mastodon/mastodon/issues/19948 https://github.com/mastodon/mastodon/issues/18692 https://github.com/mastodon/mastodon/issues/19847?

**Comment by seanthegeek** [comment]
No. The Explore page shows trending posts, not all posts from an instance.

**Comment by nemobis** [comment]
Il 23/12/22 19:47, Sean Whalen ha scritto:
> No. The Explore page shows trending posts, not all posts from an instance.

Ok, but the trending posts on one instance are not necessarily federated 
to another instance.
